examples/bri-3/test/jest-e2e.json

```diff
@@ -1,9 +1,33 @@
 {
-  "moduleFileExtensions": ["js", "json", "ts"],
-  "rootDir": ".",
-  "testEnvironment": "node",
-  "testRegex": ".e2e-spec.ts$",
-  "transform": {
-    "^.+\\.(t|j)s$": "ts-jest"
-  }
+	"compilerOptions": {
```

Comment:
i am just curious why was there a need to touch e2e tests configuration?

Reply:
this is done to future proof when we start adding integration tests.